### ISSUE TITLE
Add sensor for current electricity price without taxes

### ISSUE THREAD

**Issue by jkkataja** [T3m3z/spotprices2ha#33]
Hi,

I think it would be beneficial to have a sensor with the current electricity price without any taxes. This sensor could be used in tracking sold solar panel production.

I created one test sensor in my own fork, if you consider it good enough we can merge it to your main branch. https://github.com/jkkataja/spotprices2ha/blob/main/spot-price.yaml

**Comment by ville-juhani-honka** [comment]
Good idea. I was planning to create one of my own just for this use case so that I can more easily do calculations with solar panel production data.
